### ISSUE TITLE
Save sessions - minor enhancements

### ISSUE THREAD

**Issue by purplesky2016** [SuffolkLITLab/docassemble-AssemblyLine#276]
1. Change "Snapshot" to "Take Snapshot". This would call for action more explicitly.
2. Format Snapshot's date as `mm/dd/yyyy hh:mm`. This would make sessions without a custom label still differentiable even if ran on the same day.

**Comment by nonprofittechy** [comment]
Agreed with 1 and 2. For 3, I think adding more prefixes could be confusing. My goal with adding al_ prefix is just to prevent variable names colliding.
